homeassistant/components/pure_energie/strings.json

```diff
@@ -0,0 +1,23 @@
+{
+  "config": {
+    "flow_title": "{model} ({host})",
+    "step": {
+      "user": {
+        "data": {
+          "host": "[%key:common::config_flow::data::host%]"
+        }
+      },
+      "zeroconf_confirm": {
+        "description": "Do you want to add Pure Energie Meter (`{model}`) to Home Assistant?",
+        "title": "Discovered Pure Energie Meter device"
+      }
+    },
+    "error": {
+      "cannot_connect": "[%key:common::config_flow::error::cannot_connect%]"
+    },
+    "abort": {
+      "already_configured": "[%key:common::config_flow::abort::already_configured_device%]",
```

Comment:
We say already configured device here but in the device info we say it's a service. What is it?

Reply:
it is a device